WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -22,4 +22,14 @@ public static async Task ModifyStartupSettingAsync(bool runOnSystemStartup)
 			await MacOsStartupHelper.AddOrRemoveLoginItemAsync(runOnSystemStartup).ConfigureAwait(false);
 		}
 	}
+
+	public static bool SetAndGetCorrectStartup()
+	{
+		if (Services.UiConfig.Oobe && RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Why this was added? The default value for RunOnSystemStartup is true. This seems like a hack.

Reply:
I would do this in the UiConfig right after deserializing. The ViewModel where you currently call this might create before saving might not. 

https://stackoverflow.com/questions/18377046/net-deserialisation-with-ondeserializing-and-ondeserialized

Reply:
Also inverting the logic. The default is true in the UiConfig, but in case of linux and osx if will be changed to false but only at the first start.